### ISSUE TITLE
Generate barcode button when adding/editing item

### ISSUE THREAD

**Issue by kosimoff** [opensourcepos/opensourcepos#1965]
Someone please add barcode generating button in add/edit item page
![capture](https://user-images.githubusercontent.com/14353101/39392553-229d5846-4ad1-11e8-8df7-06d260c8616a.PNG)

**Comment by agneroalain** [comment]
Good idea, ihave thi problem for my process , but for moment , barcode is generate using information of the product .
Or maybe use a random generator for this side !

**Comment by mckaygerhard** [comment]
maybe the same process to the already generation of the itmes can be inserted there!

**Comment by daN4cat** [comment]
@agneroalain you can have barcodes generated for you by OSPOS using the item id as seed.

**Comment by SteveIreland** [comment]
I apologize if I'm way off base here.  I tend to think of the barcode as the graphical image of the UPC or EAN code.  Those codes are industry assigned.  Are you selling products that do not have an industry assigned UPC or EAN code and you're needing the system to assign a numeric code that you would use in place of the barcode?  If that's the case then wouldn't the item number that is assigned by the system to each unique item be adequate?  

Or are  you going to enter the code and you just want the button to print a barcode image to a Web page that you can then print to a printer that has a labeling sheet on it?

**Comment by mckaygerhard** [comment]
umm you have right @SteveIreland , good point!

**Comment by agneroalain** [comment]
I somoene can show me how , i want to participate to the evolution of the app
